app/governance/proposal/page.tsx

```diff
@@ -0,0 +1,315 @@
+    parseFloat(votesData.Veto),
+    parseFloat(votesData.Abstain),
+  ];
+  //console.log(Math.max(...amounts));
+
+  const maxAmountIndex = amounts.indexOf(Math.max(...amounts));
+
+  return isProposalsLoading ? (
+    <Splash />
+  ) : (
+    <div className={styles.proposalContainer}>
+      <div className={styles.proposalHeaderContainer}>
+        <div className={styles.proposalCard1}>
+          <div
+            className={styles.backButtonContainer}
+            onClick={() => {
+              router.push("/governance");
+            }}
+          >
+            <div className={styles.backButton}>
+              <Icon
+                icon={{
+                  url: "/dropdown.svg",
+                  size: 22,
+                }}
+                themed
+              />
+            </div>
+          </div>
+          <div style={{ borderRight: "1px solid", padding: "10px" }}>
+            <Text>#{proposal.proposal_id}</Text>
+          </div>
+          <div style={{ padding: "10px" }}>
+            <Text>{formatProposalStatus(proposal.status)}</Text>
+          </div>
+        </div>
+        <div style={{ padding: "10px 0px 10px 0px" }}>
+          <Text font="proto_mono" size="x-lg">
+            {proposal.title}
+          </Text>
+        </div>
+
+        <div>
+          <Text opacity={0.4}>{proposal.description}</Text>
+        </div>
+      </div>
+
+      <div className={styles.proposalCardContainer}>
+        <div className={styles.proposalCardContainer1}>
+          <div className={styles.proposalInfoBox}>
+            <div className={styles.proposalInfo}>
+              <div>
+                <Text font="proto_mono" opacity={0.3}>
+                  Type:
+                </Text>
+              </div>
+              <div>
+                <Text font="proto_mono">
+                  {formatProposalType(proposal.type_url)}
+                </Text>
+              </div>
+            </div>
+            <div className={styles.proposalInfo}>
+              <div>
+                <Text font="proto_mono" opacity={0.3}>
+                  Total Deposit:
+                </Text>
+              </div>
+              <div>
+                <Text font="proto_mono">
+                  {formatDeposit(proposal.total_deposit[0].amount)}{" "}
+                  <Image
+                    src="/tokens/canto.svg"
+                    width={16}
+                    height={16}
+                    alt="canto"
+                    style={{
+                      filter: "invert(var(--dark-mode))",
+                    }}
+                  />
+                </Text>
+              </div>
+            </div>
+            <div className={styles.proposalInfo}>
+              <div>
+                <Text font="proto_mono" opacity={0.3}>
+                  Turnout / Quorum:{" "}
+                </Text>
+              </div>
+              <div>
+                <Text font="proto_mono">
+                  {"38.1%"} &nbsp; {"33.4%"}
```

Comment:
i think we should have these as variables in a config. These could change and it will be hard to find later if we hardcode here

Reply:
Yep let's decide on where to have the constants saved and then we can change it immediately

Reply:
Done this. Now the values are in config/consts/config.ts